components/StatementsDiscussion/StatementDiscussion.js

```diff
@@ -47,16 +41,9 @@ const StatementDiscussion = ({ t, tagMappings }) => {
 
   const handleReload = e => {
     e.preventDefault()
-    const href = getFocusHref(discussion)
-    if (href) {
-      router.replace(href).then(() => {
-        refetch({
-          focusId: undefined
-        })
-      })
-    } else {
-      refetch()
-    }
+    refetch({
+      discussionId: discussion.id
+    })
```

Comment:
does this also remove the focus id? IMO we should still do that

Reply:
it does not. However I don't think this would be the expected behaviour considering the button is called "reload".
I would like to further go into how we update the url-query based on the user input. However I don't think there is enough time for that now.

Reply:
IMO this is more of a consistency issue: we're doing it this way in the other discussions and the href on the actual a tag is also without focus query. I think the behaviour with removing focus is right and how it has been everywhere in the past – see no reason to change it now.